extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -48,8 +50,8 @@ amp-story-shopping-tag {
   border-radius: 18px !important;
   height: 100% !important;
   background-color: var(--i-amphtml-shopping-tag-bg-color) !important;
-  padding: 0 6px !important;
   backdrop-filter: var(--i-amphtml-shopping-tag-backdrop-filter) !important;
+  padding: 4px 12px 4px 4px !important;
```

Comment:
RTL support please :))

Reply:
Will add RTL support once PM discussion and approval on currency format is complete. See my comment on https://github.com/ampproject/amphtml/pull/37187#discussion_r768276620

Reply:
Uneven paddings will be hard to track, I wouldn't feel comfortable merging this PR for now

Reply:
Ok, I will add an story page with an example using arabic with RTL support

Reply:
I have added rtl support and updated the description with images for comparison.

Reply:
The left padding is different than the right padding, RTL/LTR layouts won't match

Reply:
This was addressed with https://github.com/ampproject/amphtml/pull/37568